content/topics/_index.fr.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Sujets sur les systèmes distribués"
+date: Thu Jun 10 11:16:31 EDT 2021
+description: "Key topics of interest in geodistributed systems."
+draft: true
+---
+
+*Note: This list of topics of interest is constantly changing, so check back frequently for updates! Suggested readings for each topic will also be expanded and modified. Please report expired links [here](https://github.com/rotationalio/geodistributed.systems/issues).*
```

Comment:
On peut traduire en francais? Just the note, not the entire page 😉

Reply:
Let me work on it